content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -40,7 +44,9 @@ The .NET Tracer can instrument the following libraries automatically:
 | MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                            | `MongoDb`            |
 | PostgreSQL                      | `Npgsql` 4.0+                                                           | `AdoNet`             |
 
-**Note:** The ADO.NET integration instruments calls made through the `DbCommand` abstract class or the `IDbCommand` interface, regardless of the underlying implementation. It also instruments direct calls to `SqlCommand`.
+<div class="alert alert-warning">
```

Comment:
I'd consider this note more of a footnote instead of a warning so it feels unnecessary to have a yellow alert box present this information. I'd prefer leaving this as a text paragraph without extra emphasis, and I wonder if the other engineers agree with me

Reply:
I'll change it to a blue "info" note, and see what you think. Could also have it be a regular paragraph. I'm guessing some attention was added to it because people were missing it and getting confused?

Reply:
Ah yes that looks good to me, thanks for the edit! 👍🏼